week2/homework/src/index.js

```diff
@@ -1,3 +1,118 @@
 'use strict';
+let fs = require('fs');
+let order = process.argv[2];
+let orderDefinition = process.argv[3];
+let orderDefinition2 = process.argv[4];
 
-// TODO: Write the homework code in this file
+fs.readFile('./toDo.json', 'utf8', function (error, data) {
+  if (error) {
+    console.log(error);
+  }
+  else {
+    let myToDoList = JSON.parse(data);
+
+    function help() {
+      fs.readFile('./help.txt', 'utf8', function (error, data) {
+        if (error) {
+          console.log(error);
+        }
+        else {
+          console.log(data);
+        }
+      }
+      );
+    };
+
+    function list() {
+      function isEmpty(myToDoList) {
```

Comment:
Again, do not nest functions inside functions, there's no reason to do it and the code becomes harder to follow and impossible to test.
the function `isEmpty` can live everywhere in this file and it still would work :)

Reply:
Corrected